contactsummary/tests/phpunit/CRM/Contactsummary/Utils/ContractSummaryTestTrait.php

```diff
@@ -8,27 +8,27 @@
   /**
    * Creates absence period from specified start and end dates and return its ID
    *
-   * @param $startDate
-   * @param $endDate
+   * @param date $startDate in YmdHis format
+   * @param date $endDate in YmdHis format
```

Comment:
There's no date type. Since the date here is a string, you can do something like:

@param string $startDate A date string in YmdHis format